bin/test-browser.js

```diff
@@ -220,6 +220,8 @@ async function startTest() {
       }
 
       console.log('Unhandled error in test page:', err);
+      console.log('  stack:', err.stack);
+      console.log('  cause:', err.cause);
```

Comment:
```suggestion
      if("cause" in err) {
        console.log('  cause:', err.cause);
      }
```

Since it raises the min. required version by a lot:
![Error .cause compatibility](https://github.com/pouchdb/pouchdb/assets/1261013/89ea2f53-1354-4020-b477-c970fe43928f)

I would argue like @garethbowen did in [PR 8849](https://github.com/pouchdb/pouchdb/pull/8849#issuecomment-1916306988).